CHANGELOG.md

```diff
@@ -155,8 +155,14 @@ and this project adheres to
 - cosmwasm-std: Add
   `DistributionQuery::{DelegationRewards, DelegationTotalRewards, DelegatorValidators}`.
   This requires the `cosmwasm_1_4` feature to be enabled. ([#1788])
+- cosmwasm-std: Export module `cosmwasm_std::storage_keys` with
+  `namespace_with_key`, `to_length_prefixed` and `to_length_prefixed_nested` to
+  make it easier to use the strandard storage key layout documented in
+  [STORAGE_KEYS.md](https://github.com/CosmWasm/cosmwasm/blob/v1.5.0/docs/STORAGE_KEYS.md)
```

Comment:
Are you intentionally linking to v1.5.0 here? I think the content is the same as in v1.4.0, just seemed odd to me to link to 1.5 in the 1.4 changelog

Reply:
I just used the latest available tag. No strong opinion here but the content is not available in the CHANGELOG before latest main anyways.

Reply:
No strong opinion from my side either. Was just wondering.